content/svenska/data_types/genomics_transcriptomics/data.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Svensk genomik & transkriptomik data
+title: Genomik & transkriptomik data
```

Comment:
I think this one should either be "Data", or the one for protein data should be "Proteindata"

Reply:
Added _Proteindata_